docroot/themes/humsci/humsci_basic/templates/contrib/colorbox-formatter.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a formatted colorbox image field.
+ *
+ * Available variables:
+ * - image: A collection of image data.
+ * - url: An URL the image can be linked to.
+ * - attributes: Link attributes.
+ *
+ * @see template_preprocess_colorbox_formatter()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{# set a generic link label in case there is no alt attribute on the image or a caption #}
+{% set link_label = "Open image in the gallery" %}
+{# If the image has an alt attribute, use it as the link label. #}
+{% if attributes['data-cbox-img-attrs']|render != '{&quot;alt&quot;:&quot;&quot;}' %}
+  {% set link_label = "Open " ~ attributes['data-cbox-img-attrs'] ~ " image in the gallery" %}
+{% endif %}
+{% set link_label = link_label|raw %}
+{% set link_label = link_label|replace({'{&quot;alt&quot;:&quot;':'', '&quot;}':''}) %}
+
+<a href="{{ url }}" aria-controls="colorbox" role="button" aria-label="{{ link_label }}" data-cbox-img-attrs="{{ link_label }}" {{ attributes }}>{{ image }}</a>
```

Comment:
@mndonx @cienvaras I don't think we want to replace the `data-cbox-img-attrs` value with a cleaned up label. That probably still needs to be in JSON format for whatever the Colorbox plugin does with it. My guess is that is uses the value in there to set the alt attribute on the expanded image.

Reply:
The rest of the plan (setting an appropriate, cleaned up `aria-label` attribute) makes sense to me. I'll update the AC on the ticket.

Reply:
That makes sense, thanks! I'll update the PR.